page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/stepdefs/SetupSteps.java

```diff
@@ -13,31 +11,62 @@
 import ru.sbtqa.tag.pagefactory.tasks.StopVideoTask;
 import ru.sbtqa.tag.pagefactory.tasks.TaskHandler;
 
-public class SetupStepDefs {
+public class SetupSteps {
+
+    private static final ThreadLocal<Boolean> isPreSetUp = ThreadLocal.withInitial(() -> false);
+    private static final ThreadLocal<Boolean> isSetUp = ThreadLocal.withInitial(() -> false);
+    private static final ThreadLocal<Boolean> isTearDown = ThreadLocal.withInitial(() -> false);
 
-    @Before(order = 0)
     public void preSetUp(Scenario scenario) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "scenario". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)